docs/en/ingest-management/elastic-agent/configuration/autodiscovery/kubernetes-conditions-autodiscover.asciidoc

```diff
@@ -84,6 +86,13 @@ To set the target host dynamically for a targeted Pod based on its labels, use a
   condition: ${kubernetes.labels.component} == 'kube-scheduler'
 ----
 
+NOTE: Pods labels and annotations can be used in autodiscover conditions. In the case of labels or annotations that include dots(`.`), they can be used in conditions exactly as
+they are defined in the pods. For example `condition: ${kubernetes.labels.app.kubernetes.io/name} == 'ingress-nginx'`. This should not be mistaken with optional dedoted labels and annotations
```

Comment:
```This should not be mistaken with optional dedoted labels and annotations
```

This confused me more here. Do you think shall we remove it?

Reply:
Which line confused you?

Reply:
See above updated.

I mean do users need this link there.

Reply:
you mean this link `Elasticsearch(<<kubernetes-provider>>)`. I have added that as it is the page that includes all the info regarding label and annotations dedoting